simapp/app.go

```diff
@@ -372,8 +373,13 @@ func NewSimApp(
 	app.ZKAuthKeeper = *zkauthKeeper
 
 	// Fetch JWK
+	var wg sync.WaitGroup
+	wg.Add(1)
+
 	ctx := app.BaseApp.NewUncachedContext(true, tmproto.Header{})
-	app.ZKAuthKeeper.FetchJWK(ctx)
+	go app.ZKAuthKeeper.FetchJWK(ctx, &wg)
+
+	wg.Wait()
```

Comment:
Doesn't the process stop here?

Reply:
It seems like I misunderstood, but the process does not stop because it is Done after starting the goroutine with FetchJWT.
By the way, I think the goroutine in FetchJWT will start up even without this wg, what do you think?

Reply:
`go app.ZKAuthKeeper.FetchJWK(ctx, &wg)`
I mean this code probably doesn't need a goroutine.

Reply:
Is it the process of block generation? If so, I'm sure it hasn't stopped.

Reply:
Yes, it doesn't stop. That was my misunderstanding.

> By the way, I think the goroutine in FetchJWT will start up even without this wg, what do you think?
> go app.ZKAuthKeeper.FetchJWK(ctx, &wg)
> I mean this code probably doesn't need a goroutine.

What do you think about this?

Reply:
As you said, wg was unnecessary so I deleted it. thank you!